packages/app-store/campfire/api/add.ts

```diff
@@ -9,7 +10,7 @@ const handler: AppDeclarativeHandler = {
   slug: appConfig.slug,
   supportsMultipleInstalls: false,
   handlerType: "add",
-  redirectUrl: "/apps/installed",
+  redirectUrl: getInstalledAppPath({ variant: appConfig.variant, slug: appConfig.slug }),
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.